apps/src/gamelab/GameLab.js

```diff
@@ -1075,30 +1062,13 @@ GameLab.prototype.onP5Draw = function () {
  * for long enough and we have not done so already.
  */
 GameLab.prototype.captureInitialImage = function () {
-  if (!project.isOwner() || this.initialCaptureComplete || this.tickCount < CAPTURE_TICK_COUNT) {
+  if (this.initialCaptureComplete || this.tickCount < CAPTURE_TICK_COUNT) {
     return;
   }
   this.initialCaptureComplete = true;
-  captureImage();
+  captureThumbnailFromCanvas(document.getElementById('defaultCanvas0'));
```

Comment:
`captureThumbnailFromElement` measures time before capture in ms, and accepts this as an argument.
`captureThumbnailFromCanvas` measures time before capture in ticks, and leaves this check as the responsibility of the caller.

Should we try to make these two methods try to be more similar?